ConeSearch.tex

```diff
@@ -46,7 +46,16 @@
 
 
 \section*{Acknowledgments} This document was originally published as a
-document of the US National Virtual Observatory (NVO) 
+document of the US National Virtual Observatory (NVO)
+\footnote{
+	The NVO document is currently listed at
+	\url{https://sites.google.com/site/usvirtualobservatory/home/documents/nvo-documents},
+  "Conesearch definition" (Roy Williams, Bob Hanish, Alex Szalay; April
+	2002) and it is currently available at this link:
+	\url{https://drive.google.com/file/d/0BwTlZK4HpTJIWkk3VEV6Z09zOTQ/view}.
```

Comment:
I find it strange that we quote the google drive link here as the addresses are ephemeral. It is good we have the copy in github and perhaps that copy is more relevant to the standard itself and should be quoted here instead. We could then maybe have a readme in a folder containing the old versions which has the full provenance of those copies?

Reply:
I think I'll write the US-VO to see if they agree to have the NVO document preserved in the ivoa.net, someplace around the documents structure. It may be a matter of Exec approval alongside US-VO to fix this. That location would be more secure than a Google Site, however the previous official link for the NVO document disappeared after the NVO was close. Persistence is always a complex thing.